storage/client/src/main/java/com/azure/storage/common/credentials/SharedKeyCredential.java

```diff
@@ -80,30 +89,63 @@ public String generateAuthorizationHeader(URL requestURL, String httpMethod, Map
         return computeHMACSHA256(buildStringToSign(requestURL, httpMethod, headers));
     }
 
+    /**
+     * Computes a signature for the specified string using the HMAC-SHA256 algorithm.
+     * Package-private because it is used to generate SAS signatures.
+     *
+     * @param stringToSign The UTF-8-encoded string to sign.
+     * @return A {@code String} that contains the HMAC-SHA256-encoded signature.
+     * @throws InvalidKeyException If the accountKey is not a valid Base64-encoded string.
+     */
+    public String computeHmac256(final String stringToSign) throws InvalidKeyException {
```

Comment:
Should reconcile that this class has a private computeHMACSHA256 and this public computHmac256